pages/1.11/gui/services/index.md

```diff
@@ -23,3 +23,22 @@ Click the service name to open the Instances panel, which provides CPU, memory,
 For services with a web interface, hover over the service name and click ![open service](/1.11/img/open-service.png) to view it.
 
 **Tip:** You can access the Mesos web interface at `<hostname>/mesos`.
+
+# Service Status
+
+DC/OS UI introduces the following concepts to illustrate the possible status of a Service at any point in time:
+
+## Running
+The Service is reported as being successfully running.
+
+## Deploying
+Whenever the user requests a change to the Service, DC/OS is performing the required actions, which haven’t completed yet.
+
+If a Service is having difficulty getting to a Running state, a warning icon is displayed. This indicates the Service is waiting for the resource offers it requires to run or that too many tasks have failed in a short amount of time. More information can be obtained from the “Debug” section of the Service.
+
+## Recovering
+Whenever a change to the Service has been requested by DC/OS (for example if a health check has failed consecutively), the task is killed and a new task is started. Similar to Deploying, DC/OS is performing the required actions, which haven’t completed yet.
```

Comment:
I would remove the `(for example if a health check has failed consecutively)` because this is a basically a waiting stage.

**Should we use tasks or instances?** I believe we should go with instances because we use the UI and also in code.

Reply:
Good point. In this scenario we should stick to instances. Will remove the health check example.